Snakefile

```diff
@@ -416,15 +439,18 @@ rule add_electricity:
         },
         base_network="networks/" + RDIR + "base.nc",
         tech_costs=COSTS,
-        regions="resources/" + RDIR + "bus_regions/regions_onshore.geojson",
+        #regions="resources/" + RDIR + "bus_regions/regions_onshore.geojson",
         powerplants="resources/" + RDIR + "powerplants.csv",
-        load=load_data_paths,
+        #load=load_data_paths,
         #gadm_shapes="resources/" + RDIR + "shapes/MAR2.geojson", 
         #using this line instead of the following will test updated gadm shapes for MA.
         #To use: downlaod file from the google drive and place it in resources/" + RDIR + "shapes/
         #Link: https://drive.google.com/drive/u/1/folders/1dkW1wKBWvSY4i-XEuQFFBj242p0VdUlM
         gadm_shapes="resources/" + RDIR + "shapes/gadm_shapes.geojson",
         hydro_capacities="data/hydro_capacities.csv",
+        demand_profile="resources/demand.csv",
+        regions="resources/" + RDIR + "bus_regions/regions_onshore.geojson",
+        load=load_data_paths,
```

Comment:
Probably in add_electricity demand.csv is the only input needed: if the substation_lv is true for a bus name X, then demand csv will have a column named coherently to the bus X. If it is false, no column will be added and that is enough.
What do you think?
Needs to be checked if regions is only needed for the demand and nothing else.

load=load_data_paths may be removed

Moreover, in general do not move rows of inputs unless necessary (e.g. load and regions)

P.S maybe the name "demand.csv" may be revised into "demand_profiles.csv" coherently with renewable profiles).